packages/client/src/hooks/useNFTs.js

```diff
@@ -100,9 +139,18 @@ export default function useNFTs() {
     }).catch(console.error);
 
     const collections = (identifiers && identifiers[1]) ?? [];
+
     for (const identifier in collections) {
       await checkCollection(treasuryAddr, collections[identifier]);
+      return;
     }
+
+    dispatch({
```

Comment:
Why do we set the NFTs empty in the end?

Reply:
It's only setting them to empty in case the identifiers are empty -> the script that is being called will check the identifiers of the NFTs that belong to this collection, so we want to make sure the state is properly updated here.